Feat/contributor page

Sawan-Kushwah opened the pull request:
issue #234 



https://github.com/user-attachments/assets/db5df393-8bca-4d67-a3e7-1acae2d5149a


Here we you can see the real time data of contributor !!
Available in both light and dark theme !!
if UI changes need comment below i will do it

Sawan-Kushwah committed:
Added contributor page

Sawan-Kushwah committed:
some fixes in rendering

ajaynegi45 commented:
> authorize it

Kindly follow these rules https://github.com/ajaynegi45/Uttarakhand-Culture-NewUI/pull/237#pullrequestreview-2402583262

Sawan-Kushwah commented:
@ajaynegi45 which rule ?

ajaynegi45 commented:
> 🎉 Thank you @Sawan-Kushwah for your contribution! Your pull request has been submitted successfully, and a maintainer will review it soon. We’re excited to have you on board! For updates and community support, **join our [Discord](https://discord.gg/QFnvJGh72V)** and be part of the conversation. Remember to **star the repo 🌟** to help us grow, and follow [@ajaynegi45](https://github.com/ajaynegi45) to stay in the loop and increase the visibility of your contributions!

This

Sawan-Kushwah commented:
@ajaynegi45 I DONT have any discord , but i can star the repo!

Sawan-Kushwah commented:
![image](https://github.com/user-attachments/assets/625acf2b-e56d-470a-99f1-a35978cd003e)
@ajaynegi45

ajaynegi45 commented:
> i have given star to this repo

Also follow

ajaynegi45 commented:
> Sawan-Kushwah

For getting level 3. Improve the feature Functionality.

Sawan-Kushwah commented:
@ajaynegi45 can you please what specific feature want ?

ajaynegi45 commented:
> @ajaynegi45 can you please what specific feature want ?

I don't know, but for level 3 You have to add more code according to GSSOC. You can Improve UI or anything

Sawan-Kushwah commented:
@ajaynegi45  
i have added all the things for a contribution page
Added animation 
Available for both light and dark theme 
User details along with there profile link
repo status
smooth hover effects 

this is enough for an level 3 issue as in other repo they have given me level 3

ajaynegi45 commented:
> Visit Preview

Hover animation is slow. Make it little bit faster

Sawan-Kushwah committed:
faster hover effect

Sawan-Kushwah commented:
@ajaynegi45 done !!

ajaynegi45 commented:
> @ajaynegi45 i have added all the things for a contribution page Added animation Available for both light and dark theme User details along with there profile link repo status smooth hover effects
> 
> this is enough for an level 3 issue as in other repo they have given me level 3

 If anyone wants to contribute, you can show them open issues.
 
 On click contributor profile redirect to new page where you can show what they contribute to this project.

Sawan-Kushwah committed:
Added open issue section and on click open all contribution done by him

Sawan-Kushwah commented:
@ajaynegi45 
![image](https://github.com/user-attachments/assets/326a9f36-98f1-4401-973e-675e895412b9)

also added link to contributor to show them there contribution 

on click to open issue it will show all issue 
on click to contributor it will show all contributor

ajaynegi45 reviewed (changes requested):
Hi @Sawan-Kushwah,

Thank you for your prompt work on this issue. I appreciate your dedication to the project.

I have reviewed your work, and I need some changes.

- Pagination is not working properly
   

     https://github.com/user-attachments/assets/79fedc33-8ce8-447b-ad35-f1bf1cdcde95



Once again, thank you for your contribution! ❤️

Sawan-Kushwah committed:
now pagination is working well

Sawan-Kushwah committed:
Merge branch 'main' into feat/contributorPage

Sawan-Kushwah commented:
@ajaynegi45  check it once 
now pagination is working !!!

Sawan-Kushwah commented:
@ajaynegi45 WILL YOU PLEASE MERGE IT NOW ?

ajaynegi45 reviewed (changes requested):
Hi @Sawan-Kushwah,

Thank you for your prompt work on this issue. I appreciate your dedication to the project.

I have reviewed your work, and I need some changes.

-  Container are not equal in open issues and pagination is not working.
  
    <img width="1401" alt="Screenshot 2024-11-01 at 9 07 45 PM" src="https://github.com/user-attachments/assets/3a8f956b-35d8-4bc4-9050-4f1532a674bd">

- Also show star and forks list with proper paginationa

Once again, thank you for your contribution! ❤️

Sawan-Kushwah committed:
fixes issue card sizing

Sawan-Kushwah committed:
fixes issue in open issue card

Sawan-Kushwah committed:
resolved conflicts

Sawan-Kushwah commented:
@ajaynegi45 
![Screenshot 2024-11-02 153538](https://github.com/user-attachments/assets/617917a4-a2e7-4713-90ed-1ffb0d8bd26c)
i have fix its styling !!

Sawan-Kushwah commented:
@ajaynegi45 please response !!!

Sawan-Kushwah commented:
@ajaynegi45 merge my PR

Sawan-Kushwah commented:
@ajaynegi45 hey please response and merge this PR !!!
else it is of no use

Sawan-Kushwah commented:
@MastanSayyad please merge my PR else i won't any points

Sawan-Kushwah committed:
Merge branch 'main' into feat/contributorPage

ajaynegi45 reviewed (changes requested):
Hi @Sawan-Kushwah,

Thank you for your prompt work on this issue. I appreciate your dedication to the project.

I have reviewed your work, and I need some changes.   
   
at JSON.parse (<anonymous>)
at O (/vercel/path0/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:94083) {
  digest: '754964660'
}
Error occurred prerendering page "/auth". Read more: https://nextjs.org/docs/messages/prerender-error
ReferenceError: window is not defined
    at 3602 (/vercel/path0/.next/server/app/auth/page.js:1:2572)
    at t (/vercel/path0/.next/server/webpack-runtime.js:1:143)
    at 9325 (/vercel/path0/.next/server/app/auth/page.js:1:133339)
    at Object.t [as require] (/vercel/path0/.next/server/webpack-runtime.js:1:143)
    at require (/vercel/path0/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:16:18490)
    at I (/vercel/path0/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:94362)
    at /vercel/path0/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:96668
    at F._fromJSON (/vercel/path0/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:97106)
    at JSON.parse (<anonymous>)
    at O (/vercel/path0/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:94083)
   Generating static pages (12/17) 
 ✓ Generating static pages (17/17)
 Export encountered errors on following paths:
	/auth/page: /auth
Error: Command "npm run build" exited with 1


Once again, thank you for your contribution! ❤️
